Compilation error when rebasing fedora to 4.20.0-rc2

I was trying to rebase fedora to the latest RC, but I hit a problem when compiling faillog's man page. I'm assuming this is related to the latest changes in the translation process and man pages. The specific error is:
```
make[3]: *** No rule to make target 'man5/faillog.5', needed by 'all-am'.  Stop.
```

I'm attaching the complete build [build](https://github.com/user-attachments/files/29603019/build.txt) log just in case

--- a/man/cs/Makefile.am
+++ b/man/cs/Makefile.am
@@ -15,3 +15,9 @@ man_MANS = \
 	man5/shadow.5 \
 	man1/su.1 \
 	man8/vipw.8
+
+if ENABLE_LASTLOG
+man_MANS += man8/lastlog.8
+endif
+
+EXTRA_DIST = $(man_MANS)

